nucliadb_protos/knowledgebox.proto

```diff
@@ -166,3 +166,13 @@ message VectorSet {
 message VectorSets {
     map<string, VectorSet> vectorsets = 1;
 }
+
+
+message TermSynonyms {
+    repeated string synonyms = 1;
+}
+
+
+message Synonyms {
+    map <string, TermSynonyms> synonyms = 1;
```

Comment:
Maybe rename to `terms` instead of `synonyms`? I'm thinking about the client code doing:
```
for synonyms in synonyms_pb:
    for synonym in synonyms:
        ...
```